acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -17,7 +17,7 @@
   /Initializing the JRuby service/ => "JRuby didn't initialize",
   /Starting web server/ => "Expected web server to start",
   /Puppet Server has successfully started and is now ready to handle requests/ => "puppetserver never finished starting",
-  /Finished shutdown sequence/ => "Test ended without puppetserver shutting down gracefully"
+  /Beginning shutdown sequence/ => "Test ended without puppetserver triggering shutdown"
```

Comment:
Figured we could at least keep an assertion to see that the shutdown hook is getting invoked even though we're removing the check for "finished".